File: Java/Conditional Statements/Lab/07. Area of Figures/AreaOfFigures.java
import java.util.Scanner;
public class AreaOfFigures {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        //square rectangle circle triangle
        String typeOfFigure = input.nextLine();
        double area = 0;

        if(typeOfFigure.equals("square")){
            double length = Double.parseDouble(input.nextLine());
            area = length * length;

        }else if (typeOfFigure.equals("rectangle")){
            double width = Double.parseDouble(input.nextLine());
            double length = Double.parseDouble(input.nextLine());
            area = width * length;

        }else if (typeOfFigure.equals("circle")){
            double radius = Double.parseDouble(input.nextLine());
            area = Math.PI * Math.pow(radius, 2);

        }else if (typeOfFigure.equals("triangle")){
            double length = Double.parseDouble(input.nextLine());
            double height = Double.parseDouble(input.nextLine());

            area = (length / 2) * height;
        }

        System.out.printf("%.3f", area);
    }
}

File: Python/Basics/Conditional Statements/Lab/AreaOfFigures.py
from math import pi

shapeType = input()
s = None

if shapeType == "rectangle":
    a = float(input())
    b = float(input())

    s = a * b

elif shapeType == "triangle":
    aLength = float(input())
    h = float(input())

    s = aLength / 2 * h

elif shapeType == "square":
    aLength = float(input())

    s = aLength * aLength

elif shapeType == "circle":
    r = float(input())
    s = (r * r) * pi


print(f"{s:.3f}")